### ISSUE TITLE
error: You probably need to execute `npm install`

### ISSUE THREAD

**Issue by andreapavoni** [prashnts/closure-compiler-brunch#3]
Hello there, thank you for your work.

I've installed this package, but when I try to run `brunch build` I get the following error:

```
17 Sep 15:57:36 - error: Initialization error - You probably need to execute `npm install` to install brunch plugins. SyntaxError: Unexpected token =
  at /Code/node_modules/brunch/lib/plugins.js:103:17
  at Array.map (native)
  at deps.filter.dependency.map (/Code/node_modules/brunch/lib/plugins.js:91:8)
  at packages.filter.plugin.filter.plugins.map.plugin.filter.deps.filter.allPlugins.filter (/Code/node_modules/brunch/lib/plugins.js:110:19)
  at Object.packages.filter.plugin.filter.plugins.map.plugin.filter.deps.filter.exports.init.teardownBrunch [as init] (/Code/node_modules/brunch/lib/plugins.js:133:20)
  at /Code/node_modules/brunch/lib/watch.js:81:19
```

Some details:
- brunch v2.1.3
- node v5.11.1
- closure-compiler-brunch v0.1.2

Here's my `brunch.config`:

```
exports.config = {
  files: {
    javascripts: {
      joinTo: "js/app.js",
    },
    stylesheets: {
      joinTo: "css/app.css"
    },
    templates: {
      joinTo: "js/app.js"
    }
  },

  conventions: {
    assets: /^(web\/static\/assets)/
  },

  paths: {
    watched: [
      "web/static",
      "test/static"
    ],
    public: "priv/static"
  },

  plugins: {
    babel: {
      ignore: [/web\/static\/vendor/],
      presets: ['es2015', 'es2016', 'react']
    },
    closurecompiler: {
      compilationLevel: 'SIMPLE',
      createSourceMap: true
    },
    cleancss: {
      keepSpecialComments: 0,
      removeDuplicates: true,
      removeEmpty: true
    },
    gzip: {
      paths: {
        javascript: 'js',
        stylesheet: 'css'
      },
      removeOriginalFiles: false,
      renameGzipFilesToOriginalFiles: false
    }
  },

  modules: {
    autoRequire: {
      "js/app.js": [
        "web/static/js/app"
      ]
    }
  },

  npm: {
    enabled: true,
    // Whitelist the npm deps to be pulled in as front-end assets.
    // All other deps in package.json will be excluded from the bundle.
    whitelist: [],
  }
};
```

Thanks in advance for eventual help on this.

**Comment by prashnts** [comment]
Thanks for reporting this! Turns out, Google's closure compiler had a `gulp` build script, which used default function parameters -- which caused the above syntax error. They fixed this in https://github.com/google/closure-compiler-js/commit/91f5391af222387d2763595f330950d3f3b9e38f.

I am pushing a new version in a little bit, which should fix this error.

**Comment by prashnts** [comment]
I have published `v0.1.3`. Let me know if it fixed it.

**Comment by andreapavoni** [comment]
Thanks! It worked 👍
